### PR TITLE
Merge dialog views

### LINKED ISSUE #7: Merge the version selection dialogs
SelectSitVersionDialog and SelectServerVersionDialog are 99% identical so can just make one version selection dialog and pass the required data into them.

### PR DESCRIPTION
**Pull request by artehe**
We only need the one dialog view for version selection as they're nigh identical

Should resolve #7

### CODE REVIEW

**Comment by artehe** [comment]
Also should fix #6 now too as it was somewhat related.

File: SIT.Manager.Avalonia/Services/InstallerService.cs [hunk 1]
```diff
@@ -278,6 +287,76 @@ public class InstallerService(IBarNotificationService barNotificationService,
...
+
+            List<GithubRelease> result = [];
+            if (githubReleases.Any()) {
+                foreach (GithubRelease release in githubReleases) {
+                    var zipAsset = release.assets.Find(asset => asset.name.EndsWith(".zip"));
+                    if (zipAsset != null) {
+                        Match match = ServerReleaseVersionRegex().Match(release.body);
+                        if (match.Success) {
+                            string releasePatch = match.Groups[1].Value;
+                            release.tag_name = release.name + " - Tarkov Version: " + releasePatch;
```
Comment (Plootie) [inline SIT.Manager.Avalonia/Services/InstallerService.cs:309]: Can we replace this with string interpolation plz

**Review by Plootie — CHANGES_REQUESTED** [review]
Good to go once the string interpolation is added

File: SIT.Manager.Avalonia/Services/InstallerService.cs (hunk 1 above)
Comment (artehe) [reply SIT.Manager.Avalonia/Services/InstallerService.cs]: Yep done

**Review by Plootie — APPROVED** [review]

**Pull request merged**